Makefile

```diff
@@ -0,0 +1,26 @@
+BINARY := poet-ref
+DOCKER_IMAGE_REPO := poet-ref
```

Comment:
Can use `poet` instead of `poet-ref`. We'll change the project name soon, in the meanwhile we can specify the output name.